src/app.js

```diff
@@ -28,6 +30,32 @@ export function createApp() {
     app.use(require('koa-logger')());
   }
 
+  app.use(
+    reportTo([
+      {
+        group: 'cspreport',
+        maxAge: 365 * 24 * 60 * 60,
+        endpoints: [{ url: '/__cspreport__' }],
+      },
+    ])
+  );
+  app.use(
+    helmet({
+      contentSecurityPolicy: {
+        directives: {
+          defaultSrc: ["'none'"],
+          frameAncestors: ["'none'"],
+          baseUri: ["'none'"],
+          reportUri: '/__cspreport__',
```

Comment:
What is this `__cspreport__`? It it part of the Mozilla recommendations? Perhaps it would be good to link to some docs on it.

Reply:
yeah it is... look at the "web applications" part of https://github.com/firefox-devtools/profiler-server/issues/18. I'll ask if there's another reference elsewhere.

Reply:
I added a few words with a reference to the checklist too.